app/workers/feed_channel_worker.rb

```diff
@@ -0,0 +1,17 @@
+class FeedChannelWorker < ApplicationWorker
+
+  def perform(options)
+    options = options.with_indifferent_access
+    team = Team.find_by!(slack_id: options[:slack_team_id])
+    channel = team.channels.find_by!(slack_id: options[:slack_channel_id])
+    feed_channel = team.channels.find_by!(slack_id: options[:feed_slack_channel_id])
```

Comment:
Was going to go for adding some error cases here however it seems that private channels don't have the same format as public so the `SlackHelper::CHANNEL_PATTERN` doesn't properly catch private channels. Ideally this would warn when the channel wasn't joined by the bot or if the channel wasn't public

Reply:
Ahh I'm guessing you mean that private channels come through as `<#C1234567890|>` without anything actually being present in that override section, which is something I've been meaning to fix in the regex 😅 I just pushed up a fix for that bit

Reply:
Oh sweet will add the proper error states!